service/templates/projects.html

```diff
@@ -140,6 +140,17 @@ <h2> Settings for [[displayName]]</h2>
         <input class="form-control" type="text" ng-model="settingsFormData.display_name">
       </div>
     </div>
+    <div class="form-group">
```

Comment:
Can you please add a screenshot of the visible changes?

Reply:
Whoops!! thought i did that 🙈